src/molecules/BottomSheet/BottomSheet.native.js

```diff
@@ -65,123 +65,142 @@ const BottomSheetDragBar = styled(View)`
   border-radius: ${(props) => props.theme.spacings.xsmall};
 `;
 
-const BottomSheet = forwardRef(
-  (
-    {
-      snapPoint = DEFAULT_SNAP_POINT,
-      children,
-      onBackDropClick = () => {},
-      onChange = () => {},
-      onClose = () => {},
-      adjustToContentHeight = false,
-      initialHeight = 0,
-    },
-    ref,
-  ) => {
-    const theme = useTheme();
-    const [headerHeight, setHeaderHeight] = useState(0);
-    const [contentHeight, setContentHeight] = useState(0);
-    const bottomsheetChildrenGroupByType = reactChildrenGroupByType(children);
-
-    const headerComponent = bottomsheetChildrenGroupByType[BottomSheetHeader];
-    const footerComponent = bottomsheetChildrenGroupByType[BottomSheetFooter];
-    const contentComponent = bottomsheetChildrenGroupByType[BottomSheetContent];
-
-    if (headerComponent?.length > 1) {
-      throw new Error(
-        `expected to have single \`BottomSheet.Header\` but found ${headerComponent.length}`,
-      );
-    }
+const BottomSheet = ({
+  visible = false,
```

Comment:
we have default props right? why putting default values here?

Reply:
These are default props no? 🤔